Source/WebCore/dom/CustomElementDefaultARIA.cpp

```diff
@@ -91,6 +91,7 @@ RefPtr<Element> CustomElementDefaultARIA::elementForAttribute(const Element& thi
 
     RefPtr<Element> result;
     std::visit(WTF::makeVisitor([&](const AtomString& stringValue) {
+        // FIXME: This path shouldn't be necessary?
```

Comment:
I'll need to fix this one way or another, since if we want to leave this path in, it should also respect referenceTarget. However, I can't imagine how it could be exercised, so perhaps this should be a NOT_REACHED?